From d2d3019ab69f802886adc71affca84e2644de4bd Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Thu, 9 Nov 2017 12:01:14 -0800
Subject: [PATCH] Remove git hash from version

---
 src/libretro.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/libretro.c b/src/libretro.c
index 8f142ff..4014247 100644
--- a/src/libretro.c
+++ b/src/libretro.c
@@ -391,11 +391,10 @@ static int get_joystick(unsigned device)
 }
 
 const char *fuse_githash;
-static char version[] = PACKAGE_VERSION " .......";
+static char version[] = PACKAGE_VERSION;
 
 void retro_get_system_info(struct retro_system_info *info)
 {
-   memcpy(version + sizeof(PACKAGE_VERSION), fuse_githash, 7);
    info->library_name = PACKAGE_NAME;
    info->library_version = version;
    info->need_fullpath = false;
-- 
2.9.3

